core/server/master/src/main/java/alluxio/master/file/meta/InodeTree.java

```diff
@@ -666,6 +667,14 @@ public CreatePathResult createPath(LockedInodePath inodePath, CreatePathOptions<
           if (fileOptions.isCacheable()) {
             ((InodeFile) lastInode).setCacheable(true);
           }
+          MountTable.Resolution resolution = mMountTable.resolve(getPath(lastInode));
```

Comment:
@maobaolong Can you describe the strategy for maintaining journaled `UfsLastModificationTimeMs` state, including when to update the field, how to use the field, and how it behaves after restart?